Makefile

```diff
@@ -68,7 +68,7 @@ env:
 
 
 node_modules/.installed : src/js/package.json
```

Comment:
Maybe add package-lock.json as a dependency here for `npm ci`?

Reply:
Sounds reasonable. I can add it.

Reply:
Done.